tools/release_generator/release.py

```diff
@@ -148,22 +137,15 @@
 
 SliceViewer and Vates Simple Interface
 --------------------------------------
-''',
-    'mantidplot.rst':'''==================
-MantidPlot Changes
-==================
-
-.. contents:: Table of Contents
-   :local:
 
 Improvements
 ############
 
 Bugfixes
 ########
 ''',
-    'mantidworkbench.rst':'''=======================
-MantidWorkbench Changes
+    'mantidworkbench.rst':'''==================
+Workbench Changes
```

Comment:
Nearly there :) Just this final one to revert to `MantidWorkbench`

Reply:
Thanks, not sure how I missed that!